process-engine/bpmn-studio
# ✨ Distinction between ProcessList and Correlation List

## Changes

1. Add Correlation List
2. Use ProcessInstance List for Process Instances from Selected Correlation

## Issues

Closes #1769 

PR: #1888

## How to test the changes

- Open the Inspect Correlation View
- **Notice that there is a Correlation List containing all correlations for the active diagram.**
- Click on a correlation
- Change to the Process Instance List
- **Notice that the Process Instance List contains all process instances for the selected correlation.**
